cita-network/Cargo.toml

```diff
@@ -2,32 +2,34 @@
 name = "cita-network"
 version = "0.6.0"
 authors = ["Cryptape Technologies <contact@cryptape.com>"]
+edition = "2018"
 
 [dependencies]
-clap = "2"
-serde = "1.0"
-serde_derive = "1.0"
-futures = "0.1"
-byteorder = "1.0.0"
-dotenv = "0.13.0"
-bytes = "0.4"
-notify = "4.0.0"
-libproto = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
+fnv = "1.0.6"
+discovery = { git = "https://github.com/nervosnetwork/p2p.git" }
```

Comment:
Why do not use [crates.io](https://crates.io/)?
At least, add a branch name, a commit-id or something else (more detail can be found in cargo docs).

Reply:
Discovery has not released yet.

As you said, a branch name and commit-id should be added.